modules/navier_stokes/src/fvkernels/PINSFVMomentumAdvection.C

```diff
@@ -105,6 +106,9 @@ PINSFVMomentumAdvection::coeffCalculator(const Elem & elem, const ADReal & mu) c
     // Rhie-Chow coefficient for. "neighbor" is the element across the current FaceInfo (fi)
     // face from the Rhie-Chow element
 
+    const auto face_mu = _mu(std::make_tuple(fi, _cd_limiter.get(), true));
```

Comment:
if you don't pass a limiter don't you get average interpolation?

Reply:
You are now required to pass in a limiter